packages/nodes-base/nodes/Snowflake/Snowflake.node.ts

```diff
@@ -178,7 +181,12 @@ export class Snowflake implements INodeType {
 			// ----------------------------------
 
 			for (let i = 0; i < items.length; i++) {
-				const query = this.getNodeParameter('query', i) as string;
+				let query = this.getNodeParameter('query', i) as string;
+
+				for (const resolvable of getResolvables(query)) {
+					query = query.replace(resolvable, this.evaluateExpression(resolvable, i) as string);
+				}
+
```

Comment:
This is occurring so frequently that we could consider creating `getSqlQuery`.

Reply:
Makes sense, but `evaluateExpression` is part of `this` and all of the important logic is extracted in the `getResolvables`, so decide to leave this as is.